torchdata/csrc/CMakeLists.txt

```diff
@@ -31,14 +28,22 @@ if(BUILD_S3)
 
   add_library(_torchdata MODULE ${EXTENSION_SOURCES})
 
-  target_include_directories(_torchdata PRIVATE ${PROJECT_SOURCE_DIR} ${Python_INCLUDE_DIR} ${pybind11_INCLUDE_DIRS})
+  add_dependencies(_torchdata aws-cpp-sdk)
 
+  target_include_directories(
+    _torchdata
+    PRIVATE
+    ${PROJECT_SOURCE_DIR}
+    ${Python_INCLUDE_DIR}
+    ${pybind11_INCLUDE_DIRS}
+    ${aws_cpp_sdk_INCLUDE_PATH}
```

Comment:
As suggested above, change the variable name to support both system-wise aws and third-party aws

Reply:
updated.